1.load/load_profiles.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+
+import pyarrow as pa
+import pyarrow.dataset as ds
+import pyarrow.parquet as pq
+from pyarrow.dataset import DirectoryPartitioning
+import logging
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+def load_profiles(
+    dataset: str,
+    source: str,
+    batch: str = None,
+    plate: str = None,
+    columns: list = None,
```

Comment:
Running this script without defining columns (i.e. with default `columns=None`) results into an error:
```
  File "pyarrow/_dataset.pyx", line 369, in pyarrow._dataset.Dataset.to_table
  File "pyarrow/_dataset.pyx", line 2818, in pyarrow._dataset.Scanner.to_table
  File "pyarrow/error.pxi", line 144, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 100, in pyarrow.lib.check_status
pyarrow.lib.ArrowInvalid: Float value 0.0011891 was truncated converting to int64
```

Reply:
This completed successfully for me. Can you `git pull` and try again?
```
python 1.load/load_profiles.py cpg0016-jump source_4 --batch 2021_06_14_Batch6 --plate BR00124779   --output ~/Desktop/test.parquet
2023-03-29 15:00:17 INFO     Loading profiles from /cpg0016-jump/source_4/workspace/profiles/2021_06_14_Batch6/BR00124779
2023-03-29 15:00:19 INFO     Found 1 files
2023-03-29 15:00:19 INFO     Load profiles...
2023-03-29 15:05:19 INFO     Writing profiles to /Users/shsingh/Desktop/test.parquet
```

Reply:
Above command works, but this fails for me, can you try pls?
```
python 1.load/load_profiles.py cpg0016-jump source_4 --batch 2021_06_14_Batch6 --output ~/Desktop/test.parquet
```

Reply:
Shouldn't we base this on the [metadata schema](https://github.com/jump-cellpainting/datasets/tree/main/metadata)?

Reply:
Ah, so this schema is that of the `profiles` Parquet file, which is not captured in the metadata schema

We need to fix all parquet files :( 

Painful but should be quick 

https://github.com/jump-cellpainting/data-validation/pull/41

Over to @johnarevalo to verify that this is an issue, and once he approves that PR, I can run it on one batch of data to get you off the ground @alxndrkalinin

Reply:
@alxndrkalinin this is ready to go but note the different (temp) location
https://github.com/jump-cellpainting/data-validation/issues/43